File: chat-service/src/main/java/com/simonecampis/ElementalChat/dto/MessageDTO.java
package com.simonecampis.ElementalChat.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MessageDTO {
    private String id;

    private String text;

    private Date date;

    private Long idUser1;

    private Long idUser2;

    private String nomeChat;
}

File: elemental/src/dto/messagedto.ts
export class MessageDTO {

  private _id: string | undefined

  text: string

  date: Date

  idUser1: number

  idUser2: number

  nomeChat: string

  constructor(text: string,

  date: Date,

  idUser1: number,

  idUser2: number,

  nomeChat: string) {
    this.text = text
    this.date = date
    this.idUser1 = idUser1
    this.idUser2 = idUser2
    this.nomeChat = nomeChat
  }

  set id(value: string) {
    this._id = value;
  }

  get id(): string {
    return <string>this._id;
  }
}
